packages/clerk-js/src/utils/passkeys.ts

```diff
@@ -25,12 +25,18 @@ type WebAuthnGetCredentialReturn =
   CredentialReturn<__experimental_PublicKeyCredentialWithAuthenticatorAssertionResponse>;
 
 type ClerkWebAuthnErrorCode =
-  | 'passkey_exists'
-  | 'passkey_retrieval_aborted'
+  // Generic
+  | 'passkey_not_supported'
+  | 'passkeys_pa_not_supported'
+  | 'passkey_invalid_rpID_or_domain'
```

Comment:
should this be all lower case? or is it aligning with a code sent back from FAPI?

Reply:
FAPI also uses a lower case / snake case format. But these have nothing to do with FAPI as they will be thrown when something goes wrong when interacting with WebAuthN which is client only.